web/client/utils/ShareUtils.js

```diff
@@ -36,6 +36,18 @@ export const GEOSTORY_DEFAULT_SHARE_OPTIONS = {
     }
 };
 
+export const CONTEXT_DEFAULT_SHARE_OPTIONS = {
+    embedPanel: true,
+    showAPI: false,
+    embedOptions: { showTOCToggle: false, allowFullScreen: false },
+    shareUrlRegex: "(h[^#]*)#\\/context\\/([^\\/]*)\\/([A-Za-z0-9]*)",
+    shareUrlReplaceString: "$1context-embedded.html#/$3",
```

Comment:
`contexts-embedded.html` doesn't exist and it doesn't seems to be used (is it?). I think we can remove this option